kubernetes-client-api/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java

```diff
@@ -1230,4 +1234,189 @@ void givenEmptyKubeConfig_whenConfigCreated_thenShouldNotProduceNPE() throws URI
       System.clearProperty("kubeconfig");
     }
   }
+
+  @Test
+  void builder_given_severalKubeConfigsAndCurrentContextInFirstFile_then_shouldUseCurrentContextInFirstFile()
+      throws URISyntaxException {
+    try {
+      // Given
+      System.setProperty(Config.KUBERNETES_KUBECONFIG_FILES,
```

Comment:
Could you please use `kubeconfig` string literal instead in all tests? In my opinion, it's better to use string literal as the test would fail if someone changes the constant value.